### ISSUE TITLE
Can not find tenency command in php artisan

### ISSUE THREAD

**Issue by haxoman** [hyn/multi-tenant#238]
__Description__

Hi 

I got successfully published when I run 
php artisan vendor:publish --tag tenancy

then I set up for tenancy.php and webservers.php.

However when I run:
php artisan tenancy:install

I cannot find tenancy command in php artisan, and gives error as below:

[Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "tenancy" is not defined.

Did I need to clear some caches? or any tips for this?

Thanks in advance.

---

__Actual behavior__

..

__Expected behavior__

..

---
__Information__

- hyn/multi-tenant version: 3.* (as recommended at https://laravel-tenancy.com/documentation/installation)
- laravel version: 5.4
- database driver and version: mysql 5.7
- webserver software and version: apache2.4.7
- php version: php 7.1

---

__tenancy.php config__

```php

```

__webserver.php config__

```php

```

---

__Error log__

```text

```

**Comment by luceos** [comment]
What is the output of `php artisan`?

**Comment by haxoman** [comment]
I may need to reinstall? I think it is my fault, btw I will attach few images
1. https://www.awesomescreenshot.com/image/2756291/af4c1153d91dbd5d91b0e0c4bb8ba3ac
2. https://www.awesomescreenshot.com/image/2756295/b90689210d1eb9b6af14b2ac01b7de73
3. https://www.awesomescreenshot.com/image/2756302/b43ea51729b05e64ad1888a512de5058

**Comment by luceos** [comment]
The asset deployment will show a success even if no assets were deployed 🙄 . Pretty damn confusing. So did you add the service provider to your `config/app.php`?

**Comment by haxoman** [comment]
I added it in  providers = [ ...
...
...
        /*
         * Multi tenancy
         */
        // Hyn multi tenancy.
        Hyn\Tenancy\Providers\TenancyProvider::class,
        // Hyn multi tenancy webserver integration.
        Hyn\Tenancy\Providers\WebserverProvider::class,
];

and it surely created two files when I publish.
Let's just remove it and re publish it.

**Comment by haxoman** [comment]
Ah sorry it is my fault, I didn't clear config cache after I added in providers.
php artisan config:cache fix all.

**Comment by haxoman** [comment]
I know you have already explained about "system" before tenancy install,
I tried to clear it myself but hard to achieve it.

$> php artisan tenancy:install
[InvalidArgumentException]         
  Database [system] not configured

it gives that error, do i need to create new database called system? and add a system connection in config/database.php like.. as below image?

<img width="762" alt="screen shot 2017-08-24 at 10 32 31 am" src="https://user-images.githubusercontent.com/18572755/29644196-9550e63c-88b7-11e7-93ef-7b1c0ffdc910.png">

**Comment by haxoman** [comment]
Oh I got it, I move system under the connections, then all migrated.

Thanks,

**Comment by haxoman** [comment]
Sorry to bother, again,
I got this error, but I can see directory.php under vendor:

I just found directory.php line 359 uses:
/**
     * @return Website|null
     */
    public function getWebsite(): ?Website ---> this makes syntax error.
    {
        return $this->website ?: $this->environment->website();
    }
and attach a image:
<img width="1223" alt="screen shot 2017-08-24 at 12 26 33 pm" src="https://user-images.githubusercontent.com/18572755/29646621-f4394f94-88c7-11e7-9413-621b0d2e678f.png">


Not sure but I am using php7.1 and restarted a apache server to refresh my php version.
as you see my php version as below:

PHP 7.1.6-2~ubuntu14.04.1+deb.sury.org+1 (cli) (built: Jun 14 2017 05:58:40) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.1.6-2~ubuntu14.04.1+deb.sury.org+1, Copyright (c) 1999-2017, by Zend Technologies

Is php 7.1 not supporting that syntax(nullable type should be supported with 7.1)? that is why?
